### ISSUE TITLE
 provide ddl when bootstrap-start? 

### ISSUE THREAD

**Issue by chen56** [zendesk/maxwell#1028]
same like table-create:
{
   "type":"bootstrap-start",
   ...
   "sql":"create table test.e ( id int(10) not null primary key auto_increment, m double, c timestamp(6), comment varchar(255) charset 'latin1' )",
}

**Comment by osheroff** [comment]
no, that doesn't make sense to me.  can you describe what you need?

**Comment by chen56** [comment]
If bootstrap-start can give me create SQL, I can not connect the source database, I can do data isolation, and now, when I meet bootstrap-start event, you need to execute `SHOW CREATE TABLE on the source database.

**Comment by osheroff** [comment]
@chen56 can you describe your use case?  Are you trying to extract an entire table onto a different mysql server?

**Comment by chen56** [comment]
@osheroff 
I use Maxwell to synchronize some tables between two servers (source server, target server). My synchronization script belongs to `target server', only it `target server'. I don't want the synchronization script to visit `source server':

```go
//now :
func onMessage(msg MaxwellMsg){
    switch msg.Type {
	case "bootstrap-start":
             sql:= sourceMysql.run("SHOW CREATE TABLE "+msg.Database+"."msg.Table)
             targetMysql.run(sql)
        case ......
}

//i want :
func onMessage(msg MaxwellMsg){
    switch msg.Type {
	case "bootstrap-start":
             targetMysql.run(msg.sql)
        case ......
}
```
